src/ms3/bs4_parser.py

```diff
@@ -1818,6 +1818,12 @@ def make_excerpt(
         included_mcs: Tuple[int] | int,
         globalkey: Optional[str] = None,
         localkey: Optional[str] = None,
+        start_mc_onset: Optional[Fraction | float] = None,
+        end_mc_onset: Optional[Fraction | float] = None,
+        exclude_end: Optional[bool] = False,
+        user_tempo: Optional[float] = None,
```

Comment:
Please rename these to `metronome_tempo` and `metronome_beat_unit` (here and everywhere else accordingly).